packages/icons/scripts/buildConstants.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
+const path = require('path');
+// eslint-disable-next-line import/no-extraneous-dependencies
+const jsdom = require('jsdom');
+
+const svgSrcDir = path.resolve(__dirname, `../dist/svg`);
+const distFile = path.resolve(__dirname, `../dist/constants/icons.js`);
+
+const buildContants = (srcDir, file) => {
+  fs.readdir(srcDir, (err, files) => {
+    const svgs = files.filter((el) => path.extname(el) === '.svg');
+
+    if (svgs.length > 0) {
+      const icons = {};
+      let distContent = 'const icons = ';
+
+      svgs.forEach((svg) => {
+        const iconName = svg.replace('.svg', '');
+        const svgPath = path.join(srcDir, svg);
+        const svgContent = fs.readFileSync(svgPath, 'utf8');
+        const dom = new jsdom.JSDOM(svgContent);
+        const pathContent = dom.window.document.querySelector('path').getAttribute('d');
```

Comment:
```suggestion
        icons[iconName] = dom.window.document.querySelector('svg').innerHTML;
```

We should store all children, not just one path

Reply:
Ok, I've fixed this.